<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,1,7,2" columnstretch="2,2,5">
    <item row="3" column="2">
     <layout class="QGridLayout" name="Right4">
      <item row="0" column="0">
       <widget class="QTextEdit" name="txt_warning">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="4">
     <layout class="QGridLayout" name="Left2" rowstretch="1,15">
      <item row="0" column="1">
       <widget class="QLabel" name="label_shown_req">
        <property name="text">
         <string>No Requirement Shown</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QScrollArea" name="scrollArea_sections">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="section_list">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>141</width>
           <height>541</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Line" name="line_section_list">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="SectionLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="Right1">
      <item row="1" column="4">
       <widget class="QPushButton" name="btn_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="txt_search_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_total_cre">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Total Credit:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="txt_total_cre">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_input_crs">
        <property name="text">
         <string>Manually Input a Course:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="choice_semester">
        <item>
         <property name="text">
          <string>Choose a semester to add course</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year1 Fall</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year1 Spring</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year2 Fall</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year2 Spring</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year3 Fall</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year3 Spring</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year4 Fall</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year4 Spring</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="4">
     <layout class="QGridLayout" name="Left1" rowstretch="2,1,1,1,1,1,1,1,1,1,13">
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_ccc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Common Core</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ccc">
        <property name="text">
         <string>Common Core Requirement</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="choice_ccc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CCC</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_school">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show School Requirement</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QComboBox" name="choice_school">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SSCI</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_major">
        <property name="text">
         <string>Major Requirement</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="btn_major">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Major Requirement</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_school">
        <property name="text">
         <string>School Requirement</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_top">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Choose Your Requirement!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QComboBox" name="choice_major">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DSCT</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QGridLayout" name="Right3" rowstretch="0,0">
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <item row="1" column="0">
       <widget class="QScrollArea" name="scrollArea_semesters">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="main_form">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>460</width>
           <height>327</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,2,10,1,2,10,1,2,10,1,2,10" columnstretch="1,7,7">
          <item row="4" column="0" rowspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Year2</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" rowspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Year3</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_6_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="11" column="1">
           <layout class="QVBoxLayout" name="layout_sem_7"/>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_2_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="2">
           <layout class="QVBoxLayout" name="layout_sem_6"/>
          </item>
          <item row="5" column="1">
           <layout class="QVBoxLayout" name="layout_sem_3"/>
          </item>
          <item row="10" column="0" rowspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Year4</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_8_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="1">
           <layout class="QVBoxLayout" name="layout_sem_5"/>
          </item>
          <item row="10" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_7_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_3_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" rowspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Year1</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_5_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="2">
           <layout class="QVBoxLayout" name="layout_sem_4"/>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_1_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2">
           <layout class="QVBoxLayout" name="layout_sem_2"/>
          </item>
          <item row="11" column="2">
           <layout class="QVBoxLayout" name="layout_sem_8"/>
          </item>
          <item row="2" column="1">
           <layout class="QVBoxLayout" name="layout_sem_1">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
          <item row="4" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Credits:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sem_4_cre">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="Right2" stretch="2,7,7">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Year</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Fall  </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Spring      </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
